core/src/main/java/com/dooapp/fxform/controller/PropertyEditorController.java

```diff
@@ -70,8 +71,10 @@ public void changed(ObservableValue observableValue, Object o, Object o1) {
                     }
                     Object newValue = propertyElementValidator.adapt(o1, adapter);
                     // check whether the value represented in the view is different from the current model value
+                    // particular case is applied for list because two different instances of empty list are considered equals

```

Comment:
Could you extract the following duplicated code (see line 118) to an external method (e.g. isSame(oldValue, newValue)) and add unit tests covering the various cases ?

Reply:
Done!